shuup/front/templatetags/shuup_front.py

```diff
@@ -58,3 +59,11 @@ def replace_field_attrs(field, **attrs):
     for attr, value in attrs.items():
         setattr(field.field, attr, value)
     return field
+
+
+@django_jinja.library.global_function
+def get_mass_and_measurement_unit():
```

Comment:
Rename this to `get_mass_and_length_units` and add in the docstring the return type:

```
:rtype: Tuple[str, str]
```